.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: Frontend CI
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches: ['**']
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - name: Install packages
+        run: npm run install:prod:full
+
+      - name: Lint
+        run: npm run eslint && npm run stylelint && npm run svelte:check
+
+      - name: Icons
+        run: npm run fontello
+
+      - name: Build
+        run: npm run build
+
+      - name: Tests
```

please consider to place tests before build to fail faster

---

it's wrong, because tests includes idea of e2e and integration too and it requires build before Cypress run